pwndbg/heap/ptmalloc.py

```diff
@@ -94,10 +94,7 @@ def contains_chunk(self, size, chunk):
 
             # TODO: Refactor this, the bin should know how to calculate
             # largebin_index without calling into the allocator
-            if size < list(self.bins.items())[0][0]:
-                return False
             size = pwndbg.heap.current.largebin_index(size) - 64
```

Comment:
I think we should make a constant for NSMALLBINS and use it here instead of 64, so it's more clear why we're doing this.